ui/e2e/Dockerfile.sauceconnect

```diff
@@ -0,0 +1,14 @@
+FROM ubuntu
```

Comment:
Yeah, I'd tried `java:openjdk-8-jdk-alpine`, but sadly that doesn't work either :cry: